plugins/CoreAdminHome/templates/optOut.twig

```diff
@@ -69,5 +86,18 @@
     </form>
     {% endif %}
 {% endif %}
+
+{% if javascripts.external|length > 0 %}
+    {% for script in javascripts.external %}
+        <script type="text/javascript" src="{{ script|raw }}"></script>
```

Comment:
http://twig.sensiolabs.org/doc/filters/escape.html

url: escapes a string for the URI or parameter contexts. __This should not be used to escape an entire URI;__ only a subcomponent being inserted.